libs/cart/src/errors/bad-input.ts

```diff
@@ -0,0 +1,8 @@
+export class DaffBadInputError extends Error {
```

Comment:
This seems like it should go in core, but I'm not sure. Or maybe it should be named `DaffBadCartInputError`

Reply:
Yeah you're right. This will prob be used in most of the Magento drivers.

Reply:
I'll put it in `driver/magento`

Reply:
done

Reply:
I also changed the transform error function to use the one in `driver`.